tests/unit/semantic_release/hvcs/test_gitlab.py

```diff
@@ -609,7 +609,7 @@ def test_create_or_update_release_when_create_fails_and_update_fails(
     )
 
     # Execute in mocked environment expecting a GitlabUpdateError to be raised
-    with create_release_patch, edit_release_notes_patch, get_release_by_id_patch:
+    with create_release_patch, edit_release_notes_patch, get_release_by_id_patch:  # noqa: SIM117
```

Comment:
What is this about... seems like a `ruff` bug. gahh i hate using <1.0 released software.

Reply:
It's the nested `with`; the alternative they suggest is to do `with A() as a, B() as b:`, but of course that would look pretty messy here:

https://docs.astral.sh/ruff/rules/multiple-with-statements/

If the raises could be a decorator, maybe that would look better, but I think just ignoring it is probably fine here?

Reply:
Well the doc doesn't consider this situation for creation of the context manger is multiple lines and prior to the with statement.

Yes, ignore is fine here.